README.md

```diff
@@ -1,6 +1,6 @@
 # Simple S3 Resource for [Concourse CI](http://concourse.ci)
 
-Resource to upload files to S3. Unlike the [the official S3 Resource](https://github.com/concourse/s3-resource), this Resource doesn't care about files being versioned.
```

Comment:
yeah, should have mentioned:  i looked at the official resource and it seems like it doesn't care about versioning either (https://github.com/concourse/s3-resource#non-versioned-buckets)